app/src/androidTest/java/ch/epfl/sdp/drone3d/ui/auth/RegisterActivityTest.kt

```diff
@@ -102,6 +123,21 @@ class RegisterActivityTest {
             .check(matches(withEffectiveVisibility(Visibility.GONE)))
     }
 
+    @Test
+    fun loginButtonWorksWithEmptyValues() {
+        // Test that the toast is displayed
+        onView(withId(R.id.emailEditText))
+            .perform(ViewActions.typeText(""))
+        onView(withId(R.id.passwordEditText))
+            .perform(ViewActions.typeText(""))
+        onView(isRoot())
+            .perform(ViewActions.closeSoftKeyboard())
```

Comment:
Why perform typeText if no text is typed ?

Reply:
I think that you could indeed just click on the button without entering empty text since it is empty by default. However, it makes the intention clearer like this and would make the test work even if we had a default text, so I would keep it like this